roles/confluent.ksql/templates/includes/base_ksql.properties.j2

```diff
@@ -1,6 +1,6 @@
 {% set schema_registries = groups.get('schema-registry', []) %}
 
-bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{broker.config.port}}{% endfor %}
+bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{kafka_port}}{% endfor %}
 
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.